add-version-suffix/index.js

```diff
@@ -42,16 +42,14 @@ async function getCommitCount(baseBranch) {
 async function run()
 {
     let branchName = await utils.getBranchName(github);
-    if(branchName === 'dev')
-    {
-        const commitCount = await getCommitCount(branchName);
-        await exec.exec(`vc-build ChangeVersion -CustomVersionSuffix \"alpha.${commitCount}\"`).then(exitCode => {
-            if(exitCode != 0)
-            {
-                core.setFailed("vc-build ChangeVersion failed");
-            }
-        });
-    }
+    const commitCount = await getCommitCount(branchName);
```

Comment:
Can we add versionSuffix input parameter and use get-image-version.suffix value?